Exercises.ru.md

```diff
@@ -9,12 +9,12 @@
 Development Environment):
   - Можно выбрать, например, тут есть список:
   https://github.com/HowProgrammingWorks/Tools/blob/master/JavaScript/README.md
-  - У меня часто спрашивают, в чем я пишу и коказываю код. Это `mc` (midnight
-  commander), не рекомендую его всем, но может епопробовать.
+  - У меня часто спрашивают, в чем я пишу и показываю код. Это `mc` (midnight
+  commander), не рекомендую его всем, но можете попробовать.
```

Comment:
Fixed in https://github.com/HowProgrammingWorks/Introduction/commit/e63ddd63578f4756ab1c323d7e931479e0c6c6cd